### PR TITLE
zhdk reference:

### PR DESCRIPTION
**Pull request by alyssaeguavoen**
-new file
-add pictures

### CODE REVIEW

File: src/documents/referenzen/zhdk.html.eco [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+short_description: für  die ZHdK
+image_path: /images/manor/manor.jpg
+layout: project
+position: 4
+tags: ['project']
+slider:
+    -
+        image: /images/zhdk/zhdk1.jpg
+        image_alt: Der Fragebogen Generator auf erster Ebene
+        title: Der Fragebogen Generator auf erster Ebene
```
Comment (samuelsteiner) [inline src/documents/referenzen/zhdk.html.eco]: I think this title is not very well chosen, it's not customer orientated. I'd focus on solutions that got provided.

File: src/documents/referenzen/zhdk.html.eco [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        subtitle: Die Applikation ermöglicht es den Dozenten der ZHdK ohne grossen Aufwand, standardisierte Fragebogen zu erstellen!
+    -
+        image: /images/zhdk/zhdk2.jpg
+        image_alt: Der Fragebogen Generator auf zweiter Ebene
+        title: Der Fragebogen Generator auf zweiter Ebene
```
Comment (samuelsteiner) [inline src/documents/referenzen/zhdk.html.eco]: same here

File: src/documents/referenzen/zhdk.html.eco (hunk 1 above)
Comment (aminagiger) [inline src/documents/referenzen/zhdk.html.eco]: @alyssaeguavoen hier bitte die besprochenen Text einfügen

File: src/documents/referenzen/zhdk.html.eco (hunk 2 above)
Comment (aminagiger) [inline src/documents/referenzen/zhdk.html.eco]: @alyssaeguavoen same here 👍

File: src/documents/referenzen/zhdk.html.eco [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    </div>
+
+    <div class="span4">
+        <h2>Umsetzung</h2>
+
+        <div>
+            <p>Das Backend der ZHdK-Applikation wurde mit <a href="/web-entwicklung/rubyonrails/">Ruby on Rails</a> und das Frontend mit jQuery realisiert.</p>
+			      <p>Die Applikation fungiert auf zwei Ebenen. Die eine Ebene ist für die Dozierenden relevant, denn dort wird der Fragebogen erstellt. Die Fragen können in der Fragebibliothek kategorisiert, ausgesucht und mittels Klick oder “drag’n’drop” zum Fragebogen hinzugefügt werden. Alle Fragen – ausser der Standardfragen – sowie die genaue Wortwahl der Einleitung sind editierbar.
+            Auf der zweiten Ebene wird hinter den Kulissen gearbeitet, für dessen Zugang man einen Login benötigt. Standardfragen sowie zum Beispiel Einleitung und Layout/Design können dort eingefügt und bearbeitet werden. Sobald der Fragebogen fertig ist, kann er vom Ersteller als PDF-Dokument abgespeichert werden.</p>
+            <p>Eine essenzielle Besonderheit ist, dass man das heruntergeladene PDF-Dokument direkt bearbeiten kann. Somit ist auch gewährleistet, dass sich die Darstellung des PDFs nicht verändert.</p>
```
Comment (samuelsteiner) [inline src/documents/referenzen/zhdk.html.eco]: I don't really get this?

File: src/documents/referenzen/zhdk.html.eco [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+    </div>
+
+    <div class="span4">
+        <h2>Umsetzung</h2>
+
+        <div>
+            <p>Das Backend der ZHdK-Applikation wurde mit <a href="/web-entwicklung/rubyonrails/">Ruby on Rails</a> und das Frontend mit jQuery realisiert.</p>
+			      <p>Die Applikation fungiert auf zwei Ebenen. Die eine Ebene ist für die Dozierenden relevant, denn dort wird der Fragebogen erstellt. Die Fragen können in der Fragebibliothek kategorisiert, ausgesucht und mittels Klick oder “drag’n’drop” zum Fragebogen hinzugefügt werden. Alle Fragen – ausser der Standardfragen – sowie die genaue Wortwahl der Einleitung sind editierbar.
+            Auf der zweiten Ebene wird hinter den Kulissen gearbeitet, für dessen Zugang man einen Login benötigt. Standardfragen sowie zum Beispiel Einleitung und Layout/Design können dort eingefügt und bearbeitet werden. Sobald der Fragebogen fertig ist, kann er vom Ersteller als PDF-Dokument abgespeichert werden.</p>
```
Comment (samuelsteiner) [inline src/documents/referenzen/zhdk.html.eco]: einen Login / ein Login?

**Pull request merged**